packages/docs/src/pages/color-modes.mdx

```diff
@@ -28,13 +30,112 @@ In the `theme.colors` object, add a nested `modes` object that will contain keys
 }
 ```
 
-The colors defined at the root of the `colors` object will be accessible as `default`. All colors
-found in `colors.modes` will be referenced by their key. The above example will have two modes: `default` and `dark`.
+The colors defined at the root of the `colors` object will be accessible as
+`default`. All colors found in `colors.modes` will be referenced by their key.
+The above example will have two modes: `default` and `dark`.
+
+## Colors Object
+
+The `colors` object contains Custom CSS Properties
+
+```js
+{
+  colors: {
+    text: 'var(--theme-ui-colors-text)',
+    background: 'var(--theme-ui-colors-background)',
+    primary: 'var(--theme-ui-colors-primary)',
+  }
+}
+```
+
+## rawColors Object
+
+If you need to pass original value somewhere where CSS Properties (e.g. WebGL
+Canvas) won't work, use `rawColors`
+
+```js
+{
+  rawColors: {
+    text: '#000',
+    background: '#fff',
+    primary: '#07c',
+  }
+}
+```
+
+## Default colors
+
+The default colors are tranformed upon `colorMode` change. This allows for the
```

Comment:
I'm kinda skeptical towards the phrase "default colors" as it can be confused with `primary.__default`. Could we say "initial color mode"?

Reply:
@lachlanjc what do you think about it?